<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Led</class>
 <widget class="QWidget" name="Led">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="onButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="led8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LED8</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="all_on">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL ON</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="all_off">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL OFF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
